hack/update: kicbase version

prezha opened the pull request:
fixes: #9464
fixes: #9420
replaces: #9439

pr #9464 ("automate update processes") split: part 4/4 includes: `hack/update/kicbase_version/update_kicbase_version.go`

<!-- 🎉 Thank you for contributing to minikube! 🎉 Here are some hints to get your PR merged faster:

1. Your PR title will be included in the release notes, choose it carefully
2. If the PR fixes an issue, add "fixes #<issue number>" to the description.
3. If the PR is a user interface change, please include a "before" and "after" example.
4. If the PR is a large design change, please include an enhancement proposal:
https://github.com/kubernetes/minikube/tree/master/enhancements
-->

sharifelgamal commented:
I think in breaking up your update PRs, some sort of dependency broke, is there a PR blocking this one?

prezha commented:
the conflicts are due to the pr #9464 (the original one - before i've split it into several prs) got merged first, and i had a couple of changes made in those individual prs; essentially my fault, i should have had closed the original pr right after the split (but left it just in case anyone wanted to compare)...
i've fixed this one now, should be ok to merge

tstromberg commented:
Thank you!
